x/supply/internal/types/account.go

```diff
@@ -1,19 +1,32 @@
 package types
 
 import (
+	"errors"
 	"fmt"
+	"strings"
 
 	yaml "gopkg.in/yaml.v2"
 
 	"github.com/tendermint/tendermint/crypto"
 
 	sdk "github.com/cosmos/cosmos-sdk/types"
+	authexported "github.com/cosmos/cosmos-sdk/x/auth/exported"
 	authtypes "github.com/cosmos/cosmos-sdk/x/auth/types"
 	"github.com/cosmos/cosmos-sdk/x/supply/exported"
 )
 
+const (
+	// ModuleAccountType defines the type for a ModuleAccount
+	ModuleAccountType = "ModuleAccount"
+)
+
+var _ authexported.GenesisAccount = (*ModuleAccount)(nil)
 var _ exported.ModuleAccountI = (*ModuleAccount)(nil)
 
+func init() {
```

Comment:
Don't think we need to do this as the `ModuleAccount` is not intended to be a concrete implementation of `GenesisAccount`.

Reply:
is this not needed to be able to import module accounts from genesis (say after an export)?